### DISCUSSION TITLE
QEMU errorexited with code-1:(no message) Windows7

### DISCUSSION THREAD

**Discussion by CAPITAN12345** [utmapp/UTM#6365] [UTM Help (iOS)]
**Describe the issue**
After many tries of trying to installing Windows 7 direct to my iPad Air 5 Generation the UTM SE, found that if I create the VM in the Mac first and then make a copy and pasted to the iPad it works correct. 
but wen I try to do the same in the iPad 9th generation I get the next message 
QEMU error: QEMU exited from an error: 2024-05-16 20:58:38.258 UTM SE [816:65455]
[QEMULogging(0x300722d00)] QEMU exited with code -1: (no message)

[debug.log](https://github.com/utmapp/UTM/files/15343174/debug.log)

**Configuration (required)**
* UTM Version: UTM SE
* OS Version: iPadOS Version 17.5
* Device Model: iPad (9th generation)
* Is it jailbroken (name jailbreak used)? NO
* How did you install UTM?  Using AltStore

**Comment by osy** [comment]
Usually this means memory allocation has failed. Try lowering the amount of memory.

**Comment by CAPITAN12345** [comment]
Yes that was the error the space that I give to the UTM thank you.

**Comment by nbbbbbbbb8** [comment]
lol

**Comment by nbbbbbbbb8** [comment]
Mind as well close it
